lib/std/target/aarch64.zig

```diff
@@ -2252,4 +2252,19 @@ pub const cpu = struct {
             .v8a,
         }),
     };
+
+    pub const microsoft_sq3 = CpuModel{
+        .name = "microsoft_sq3",
+        .llvm_name = "generic",
```

Comment:
```suggestion
        .llvm_name = "null",
```

(but this will go into tools/update_cpu_features.zig which generates this file)

Reply:
Ah, so it's referred to as `null` when unknown rather than `generic`, correct?

Reply:
Yes; ultimately it will do the same thing, but it does communicate to Zig's system that there is not actually a corresponding LLVM name.

Reply:
If it's too much trouble, feel free to commit these changes as-is and when I go and run the tools/update_cpu_features.zig script in the future, I'll notice the diff deleted this new CPU and add it back to the script at that time.

Reply:
Actually, I wanna investigate a little more. I noticed there is a bug in pulling the records from the registry in that it was only pulling the last cpu which turns out `big` from `big.LITTLE`, and we might not even need to add any new cpus after all as Snapdragon 8cx gen3 is based on A78 (LITTLE) and X1 (big).

Reply:
There is no need to update the CPU features just yet as the detected core models are already tabulated: `LITTLE` A78C and `big` X1C. I have reflected that in the cpu model table in `system/arm.zig` with a note that `0xD4C` is most likely an `X1C` core, however this should still be verified one way or another. My guess is based on the description of [Microsoft SQ3 / Qualcomm Snapdragon 8cx Gen 3](https://www.notebookcheck.net/Qualcomm-Snapdragon-8cx-Gen-3-Processor-Benchmarks-and-Specs.652916.0.html).